Jenkinsfile

```diff
@@ -71,12 +71,35 @@ pipeline {
 
     stage('Parallel stage') {
       parallel {
-        stage('Compile and test memcpy_loop') {
+        stage('Compile and test microbenchmarks') {
+          options { timeout(time: 5, unit: 'MINUTES') }
```

Comment:
I see the last successful run took 24s. 
* Do you envision it ever hitting this timeout? I mean, more tests obviously take more time.
* And/or was it just randomly chosen at 5 minutes?

Reply:
I don't expect it to time out, but a breakage could lead to something getting totally wedged and hanging.

I just noticed that we'd never supplied a timeout, so I copied the one used for building documentation.